pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -695,3 +695,22 @@ def test_loc_getitem_index_differently_ordered_slice_none():
         columns=["a", "b"],
     )
     tm.assert_frame_equal(result, expected)
+
+
+def test_loc_empty_multiindex():
```

Comment:
can you co-locate this with similar tests

```
jreback@lhs2:~/pandas-dev$ grep empty pandas/tests/indexing/multiindex/*
pandas/tests/indexing/multiindex/test_getitem.py:def test_frame_getitem_multicolumn_empty_level():
pandas/tests/indexing/multiindex/test_getitem.py:def test_frame_mi_empty_slice():
pandas/tests/indexing/multiindex/test_loc.py:        empty = Series(data=[], dtype=np.float64)
pandas/tests/indexing/multiindex/test_loc.py:        result = x.loc[empty]
pandas/tests/indexing/multiindex/test_loc.py:        # empty array:
pandas/tests/indexing/multiindex/test_loc.py:        empty = np.array([])
pandas/tests/indexing/multiindex/test_loc.py:        result = x.loc[empty]
pandas/tests/indexing/multiindex/test_loc.py:        ([], []),  # empty ok
pandas/tests/indexing/multiindex/test_loc.py:def test_loc_getitem_duplicates_multiindex_empty_indexer(columns_indexer):
pandas/tests/indexing/multiindex/test_loc.py:    # empty indexer
```

IOW there are 2 tests in the test_getitem.py that are testing basically the same, so locate there.

Reply:
Moved the test to *pandas/tests/indexing/multiindex/test_getitem.py*